kasa/smart/modules/time.py

```diff
@@ -83,3 +84,12 @@ async def set_time(self, dt: datetime) -> dict:
             if region:
                 params["region"] = region
         return await self.call("set_device_time", params)
+
+    async def _check_supported(self):
+        """Additional check to see if the module is supported by the device.
+
+        Hub attached sensors report the time module but do retun device time.
```

Comment:
```suggestion
        Hub attached sensors report the time module but do not implement it.
```
Perhaps? I'd probably ship this specific change as its own PR as it affects already existing code.